src/main/java/domain/Cash.java

```diff
@@ -0,0 +1,28 @@
+package domain;
+
+public class Cash {
+
+	private final int value;
+
+	public Cash(int value) {
+		validateNegative(value);
+		validateLessThanTicketPrice(value);
+		this.value = value;
+	}
+
+	private void validateNegative(int value) {
+		if (value < 0) {
+			throw new IllegalArgumentException("Cash의 값은 음수가 아니여야 합니다.");
```

Comment:
예외 메시지로 입력받은 값을 보여준다면 조금 더 에러를 잡기가 쉬울 것 같아요. 🙂
```suggestion
			throw new IllegalArgumentException("Cash의 값은 음수가 아니여야 합니다. value: " + value);
```

Reply:
반영하였습니다.
https://github.com/next-step/java-lotto-kakao/pull/14/commits/d67f76c809c99e7e7b95cb28e791784b329be250